.github/comment.yml

```diff
@@ -105,7 +126,10 @@
 - rule:
     type: label
     label: ARMChangesRequested
-    onLabeledComments: "Please ensure to respond feedbacks from the ARM API reviewer. When you are ready to continue the ARM API review, please remove `ARMChangesRequested`"
+    onLabeledComments: >-
+      Please ensure to respond to feedback from the ARM API reviewer.
+      When you are ready to continue the ARM API review, please remove the `ARMChangesRequested` label.
+      This will notify the reviewer to have another look.
```

Comment:
>look [](http://example.com/codeflow?start=52&length=4)

If the feedback provided needs further discussion, please use this Teams channel to post your questions - [https://aka.ms/swaggersupport](https://aka.ms/swaggersupport)

Reply:
I will ask the API reviewer on-call to keep a tab on this teams channel going forward.

Reply:
@rkmanda I have applied the changes. Today on standup myself, @maririos and @kurtzeborn were discussing if we should have separate channel for ARM reviews. Mariana and myself lean towards keeping this channel for both ARM and data plane reviews, instead of introducing a new one. This is to avoid "channel sprawl". I already had to redirect 5+ people from the "general" channel to the "API Spec Review" channel.

Reply:
makes sense to me to have a single channel and I will have ARM API reviewers keep a tab on this channel for questions related to ARM

Reply:
@rkmanda, in our meeting two weeks ago it was agreed that we would separate the Teams channels into one focused on support for PRs changing data plane specs, and another channel focused on support for PRs changing control plane specs.  @konrad-jamrozik was not at this meeting, but we all agreed that this was acceptable and probably the best, clearest path forward.

@rkmanda, please provide us with the location of the ARM Teams channel for supporting PRs which change control plane specs so that we can update the docs and bot messages to point to that location.  If we don't have one by the end of the month (when the vendors transition to other work), we can just send them to the alias you created which I believe you said includes all of your leads.